config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Shhhhh. These are secrets. They shouldn't float around out in the open. Pop this file in your gitignore.